.azure/pipelines/jobs/default-build.yml

```diff
@@ -41,217 +41,223 @@
-    cancelTimeoutInMinutes: ${{ parameters.cancelTimeoutInMinutes }}
-    ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.agentOs, 'Windows'), eq(parameters.codeSign, 'true')) }}:
-      enableMicrobuild: true
-      enablePublishBuildAssets: true
-      enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
-    enablePublishTestResults: true # publish test results to AzDO (populates AzDO Tests tab)
-    enableTelemetry: true
-    helixRepo: dotnet/aspnetcore
-    helixType: build.product/
-    workspace:
-      clean: all
-    # Map friendly OS names to the right queue
-    # See https://github.com/dotnet/arcade/blob/master/Documentation/ChoosingAMachinePool.md
-    pool:
-      ${{ if eq(parameters.agentOs, 'macOS') }}:
-        vmImage: macOS-10.13
-      ${{ if eq(parameters.agentOs, 'Linux') }}:
-        vmImage: ubuntu-16.04
-      ${{ if eq(parameters.agentOs, 'Windows') }}:
-        ${{ if eq(variables['System.TeamProject'], 'public') }}:
-          name: NetCorePublic-Pool
-          ${{ if ne(parameters.isTestingJob, true) }}:
-            # Visual Studio Build Tools
-            queue: BuildPool.Server.Amd64.VS2019.BT.Open
-          ${{ if eq(parameters.isTestingJob, true) }}:
+  - template: /eng/common/templates/job/job.yml
+    parameters:
+      name: ${{ coalesce(parameters.jobName, parameters.agentOs) }}
+      displayName: ${{ coalesce(parameters.jobDisplayName, parameters.agentOs) }}
+      dependsOn: ${{ parameters.dependsOn }}
+      ${{ if ne(parameters.condition, '') }}:
+        condition: ${{ parameters.condition }}
+      timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+      cancelTimeoutInMinutes: ${{ parameters.cancelTimeoutInMinutes }}
+      ? ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.agentOs, 'Windows'), eq(parameters.codeSign, 'true')) }}
+      : enableMicrobuild: true
+        enablePublishBuildAssets: true
+        enablePublishUsingPipelines: ${{ variables._PublishUsingPipelines }}
+      enablePublishTestResults: true # publish test results to AzDO (populates AzDO Tests tab)
+      mergeTestResults: true
+      testRunTitle: ${{ parameters.testRunTitle }}
+      enableTelemetry: true
+      helixRepo: dotnet/aspnetcore
+      helixType: build.product/
+      workspace:
+        clean: all
+      # Map friendly OS names to the right queue
+      # See https://github.com/dotnet/arcade/blob/master/Documentation/ChoosingAMachinePool.md
+      pool:
+        ${{ if eq(parameters.agentOs, 'macOS') }}:
+          vmImage: macOS-10.13
+        ${{ if eq(parameters.agentOs, 'Linux') }}:
+          vmImage: ubuntu-16.04
+        ${{ if eq(parameters.agentOs, 'Windows') }}:
+          ${{ if eq(variables['System.TeamProject'], 'public') }}:
+            name: NetCorePublic-Pool
+            ${{ if ne(parameters.isTestingJob, true) }}:
+              # Visual Studio Build Tools
+              queue: BuildPool.Server.Amd64.VS2019.BT.Open
```

Comment:
Suggestion: Unrelated to this PR, but please consider just using Enterprise for either build; since this was primarily in place to be a non-expiring SKU from before RTW, it'd be nice to eventually obsolete buildtools skus since we won't be able to ever obsolete enterprise and it should be able to build buildtools.

Reply:
This is only showing as edited because of the indentation change. I'd suggest filing a new issue to discuss that.